recipes/iscan/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "iscan" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f1b209154df73c67743e53b203fe41c354a079cdd943018a2f84aa642f27dd8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
```suggestion
    - python >=3.5
```

Reply:
I understand it's necessary to specify this range in the `run` section, but could you explain why it's necessary to include this in the `host` section as well? My understanding is that the only place where python will come into play in the build process is during the following step. Would the version matter?
```
script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
The host and run should be running the same python to avoid any problem